packages/test/test-end-to-end-tests/src/test/mapEndToEndTests.spec.ts

```diff
@@ -341,9 +341,6 @@ describeNoCompat("SharedMap orderSequentially", (getTestObjectProvider) => {
     beforeEach(() => {
         provider = getTestObjectProvider();
     });
-    afterEach(async () => reset());
-
-    const reset = async () => provider.reset();
```

Comment:
reset will can cause a failure due to logging errors when called directly. however, the provider itself will automatically reset in a way that will not duplicate failures, so it is usually unnecessary to call.

Reply:
Since you wrote "usually," when is it necessary to call?

Reply:
when you are not in a describeNoCompat (like nested describe) and want to provider to reset. This is because the auto-reset happens in the after each of the describeNoCompat, so nested descibe compats with after eachs won't reset automatically